### ISSUE TITLE
🐛  [2.0 Beta] Language Server Crashes

### ISSUE THREAD

**Issue by thejackshelton-kunaico** [biomejs/biome-vscode#543]
### VS Code version

1.98.2

### Extension version

2025.3.151143 (pre-release)

### Biome version

2.0.0-beta.1

### Operating system

- [ ] Windows
- [x] macOS
- [ ] Linux

### Description

When a plugin is enabled, the language server crashes on a simple lint check from a grit file.

The MRE below is a minimal Qwik application that has one grit query and one route.

### Steps to reproduce

1. Go to https://github.com/thejackshelton/biome-plugin/tree/main
2. Install the pre-release biome plugin in VSCode
3. Run pnpm i
4. Exit and enter the VSCode app OR reload window
5. Navigate to index.tsx
6. You will see language server crashes from the lib.rs file

### Expected behavior

The language server should not crash.

### Does this issue occur when using the CLI directly?

Not sure / Not applicable

### Link to a minimal reproduction

https://github.com/thejackshelton/biome-plugin/tree/main

### Logs

```plain text
Stack trace:


2025-03-26 10:56:54.620 [info] Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: /Users/runner/work/biome/biome/crates/biome_analyze/src/lib.rs:196:50
Thread Name: tokio-runtime-worker
Message: missing span


2025-03-26 10:56:54.642 [info] [Error - 10:56:54 AM] Biome language server closed
```
